test/cmdlineTests/asm_json/output

```diff
@@ -450,9 +450,9 @@ EVM assembly:
         {
           "begin": 77,
           "end": 158,
+          "jumpType": "[in]",
```

Comment:
hmm.. I guess that should not be like that..

Reply:
Is this comment still relevant?